src/Test/WordPress/HooksTest.php

```diff
@@ -42,9 +42,10 @@ public function setup()
         $this->mockDefaultIntegration = new DefaultIntegration($this->mockConfig, $this->mockWordPressAPI, $this->mockDataStore, $this->mockLogger);
 
         $this->hooks = $this->getMockBuilder('\CF\WordPress\Hooks')
-            ->setMethods(array('setHooks'))
+            ->disableOriginalConstructor()
+            ->setMethods(array('__construct')) // This is a hack to make the tests work
```

Comment:
When you write tests for a class you don't mock the class that is being tested.  Since this is HooksTest.php we shouldn't be mocking Hooks, we should be instantiating the real one.